src/store/Account.ts

```diff
@@ -389,6 +396,20 @@ export default {
                 )
                 .toPromise();
 
+            // sorted array to be represented in multisig tree
+            const getMultisigAccountGraphArraySortedPromise = repositoryFactory
```

Comment:
Why are we making the same API call here again? Is it just to get the result as sorted? If so why not we use the previous result and sort it?

Reply:
makes sense